langchain/scripts/create-entrypoints.js

```diff
@@ -40,6 +40,7 @@ const entrypoints = {
   "embeddings/tensorflow": "embeddings/tensorflow",
   "embeddings/hf": "embeddings/hf",
   "embeddings/googlevertexai": "embeddings/googlevertexai",
+  "embeddings/googlevertexai-image": "embeddings/googlevertexai-image",
```

Comment:
Underscore is convention vs. hyphen

Reply:
Will change, tho this was based on llms/openai-chat